packages/accounts-passwordless/package.js

```diff
@@ -0,0 +1,22 @@
+Package.describe({
+  summary: 'No-password login/sign-up support for accounts',
+  version: '0.0.1',
```

Comment:
Should be v1.0.0 or will be releasing it as an experimental package for now?

Reply:
I prefer going with 1.0.0 as well.